qiskit/tools/visualization/_circuit_visualization.py

```diff
@@ -192,8 +192,13 @@ def circuit_drawer(circuit,
                                             style)
     else:
         if output == 'text':
+            if style:
+                reversebits = style.get('reversebits', False)
```

Comment:
CI is a better reviewer than me. We need to define a default for `reversebits` and `plotbarriers` because in the case we don't define style these variables won't ever be defined  for when we call `_text_circuit_drawer()` below at L199